packages/xo-server-sdn-controller/src/index.js

```diff
@@ -760,7 +723,8 @@ class SDNController extends EventEmitter {
       const hosts = filter(pool.$xapi.objects.all, { $type: 'host' })
       await Promise.all(
```

Comment:
it could be a good idea to use asynceach here even if it's not related directly to this PR , since there can be a lot of hosts